cl/simple_pages/templates/help/advanced_search.html

```diff
@@ -51,17 +57,27 @@ <h3>Grouped Queries and subqueries: <code>( )</code></h3>
   <p>Using parentheses will group parts of a query (e.g. <code>(customs OR "border patrol") AND asylum</code>). Parentheses can be nested as deeply as needed.
   </p>
 
-  <h3>Wildcards and Fuzzy Search: <code>*</code>, <code>?</code>, and <code>~</code></h3>
-  <p>Using an asterisk (<strong>*</strong>) allows wildcard searches. For example, <code>immigra*</code> finds all words beginning with "immigra". This can also be used at the beginning or middle of words, at both the beginning and the end of a word, or even all three. For example, you can find all words containing two esses side-by-side with the following query: <code>*ss*</code>. You could also find words with two esses separated by other letters with a query such as: <code>*s*s*</code>. This would find cases containing words like "<strong>s</strong>u<strong>s</strong>an" or   "as<strong>s</strong>i<strong>s</strong>tant".
-  </p>
+  <h3>Wildcards and Fuzzy Search: <code>*</code>, <code>!</code>, <code>?</code> and <code>~</code></h3>
+  <p>Using an asterisk (<code>*</code>) allows for wildcard searches. For example, <code>immigra*</code> finds all words that begin with "immigra". Alternatively, you can use an exclamation mark (<code>!</code>) at the beginning of a word for the same purpose. For instance, <code>!immigra</code> matches words that start with "immigra".</p>
 
-  <p>The question mark character (<strong>?</strong>) can be used similarly as a single letter wildcard. For example, this would find cases containing the word "immigrant" or "emmigration": <code>?mmigra*</code>
-  </p>
+  <p><code>*</code> can also be used inside words, where it acts as a single-character wildcard. For example, a query like <code>gr*mm*r</code> would match cases containing both "gr<strong>a</strong>mm<strong>a</strong>r" and "gr<strong>i</strong>mm<strong>e</strong>r".</p>
```

Comment:
I think we need `<nobr>`'s here, right? 

```suggestion
  <p><code>*</code> can also be used inside words, where it acts as a single-character wildcard. For example, a query like <nobr><code>gr*mm*r</code></nobr> would match cases containing both <nobr>"gr<strong>a</strong>mm<strong>a</strong>r"</nobr> and <nobr>"gr<strong>i</strong>mm<strong>e</strong>r"</nobr>.</p>
```

Reply:
I found that `<nobr>` is deprecated and is no longer recommended for use.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nobr

Instead used `text-nowrap` css class.